main.py

```diff
@@ -44,6 +44,9 @@ def main():
             with open("config.json", 'w') as file:
                 json.dump(configuration, file, sort_keys=True, indent=4, ensure_ascii=False)
 
+        player_max = configuration["player_max"]
```

Comment:
The new configuration values introduce changes, which could break older versions of the software (due to missing configuration values, which will crash the program with a KeyError)
You may get the value from the configuration via `configuration.get("player_max", 10)` where 10 is a default value in case, the key does not exist in the dict

This does apply for lines 48 and 49 as well